docs/network/edges.html

```diff
@@ -895,7 +899,31 @@ <h3>Options</h3>
             <td>selfReferenceSize</td>
             <td>Number</td>
             <td><code>false</code></td>
-            <td>When the to and from nodes are the same, a circle is drawn. This is the radius of that circle.</td>
+            <td>When the to and from nodes are the same, a circle is drawn. This is the radius of that circle. 
+                This property is deprecated please use selfReference instead.
+            </td>
+        </tr>
+        <tr class='toggle collapsible' onclick="toggleTable('optionTable','selfReference', this);">
+            <td><span parent="arrows" class="right-caret"></span> selfReference</td>
+            <td>Object</td>
+            <td><code>Object</code></td>
+            <td>When the to and from nodes are the same, a circle is drawn. This provides radius of circle and position.
+            </td>
+        </tr>
+        <tr parent="selfReference" class="hidden">
+            <td class="indent">size</td>
+            <td>Number</td>
+            <td><code>false</code></td>
```

Comment:
The default value for datatype number cannot be false. In the code it was and is 20.